### PR TITLE
Create a React component that matches the Maths Magicians UI design

### PR DESCRIPTION
**Pull request by Tekhlay**
### Milestones 🥇 

- [x] Delete all the default boilerplate from CRA (text, images, styles).
- [x] Create a directory called components.
- [x] create a new Calculator.js file inside the cominsideponents folder.
- [x] create a React component that matches the maths magicians UI design:

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,14 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (gemmen29) [inline src/App.js]: - [ ] Please remove the `eslint-disable` line and try to fix it manually as this defies the purpose of using linters in the first place.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/button-has-type */
+import React, { Component } from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (gemmen29) [inline src/components/Calculator.js]: - [ ] Please remove the `eslint-disable` line and try to fix it manually as this defies the purpose of using linters in the first place.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Tekhlay, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Teklay, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**